jenkins-master/jobs/scripts/workspace/config.py

```diff
@@ -29,9 +29,9 @@
             'harness_config': os.path.join('firefox_ui_tests', 'qa_jenkins.py'),
             'harness_script': os.path.join('firefox_ui_tests', 'update.py'),
             'treeherder': {
-                'group_name': 'Firefox UI Tests - update',
-                'group_symbol': 'Fu',
-                'job_name': 'Firefox UI Tests - update ({locale}-{update_number})',
+                'group_name': 'Firefox UI Update Tests - {update_channel}',
+                'group_symbol': 'Fu-{update_channel}',
```

Comment:
You realize that some people reading "Fu" in English will associate with an obscene expression? "F#$# you!"